app/scripts/views/settings.js

```diff
@@ -206,6 +206,9 @@ function ($, modal, Cocktail, Session, BaseView, AvatarMixin,
         })
         .fin(function () {
           self.logViewEvent('signout.success');
+          // Unset uid otherwise it will henceforth be impossible
+          // to sign in to different accounts inside this tab.
+          self.relier.unset('uid');
           self.user.clearSignedInAccount();
```

Comment:
Do you mean just `unset()` those two specifically or should I `clear()` the whole object?